### ISSUE TITLE
Xcode 13 RC build error

### ISSUE THREAD

**Issue by janremes** [apollographql/iOSCodegenTemplate#4]
Hi, we are seeing build error when using Xcode 13 and generating schema using ApolloCodegen
in PathKit.swift on line 583

```
value of optional type 'UnsafeMutablePointer<CChar>?' (aka 'Optional<UnsafeMutablePointer<Int8>>') must be unwrapped to a value of type 'UnsafeMutablePointer<CChar>' (aka 'UnsafeMutablePointer<Int8>')
```

<img width="1176" alt="Screenshot 2021-09-17 at 17 53 06" src="https://user-images.githubusercontent.com/1276036/133817823-f58dfd9b-0734-4020-baf5-1c27d072ccb5.png">

Please can you reproduce it?

**Comment by janremes** [comment]
sorry wrong place to report this: https://github.com/kylef/PathKit/issues/77

**Comment by morgz** [comment]
Pathkit is a dependency of this project, so won't we want to keep this issue open to make sure the dependency is updated? I'm also having this issue with Xcode RC 13.

**Comment by calvincestari** [comment]
Thanks @janremes, @morgz - looks like this was fixed in [PathKit `1.0.1`](https://github.com/kylef/PathKit/releases/tag/1.0.1); I'll make sure our dependency versioning pulls that in.
